generator/templates/functions/template.m.jinja2

```diff
@@ -6,16 +6,13 @@
 #import "SDLRPCParameterNames.h"
 {%- endblock %}
 {% block constructors %}
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
This #pragma push/pop needs to be here because `initWithName` was deprecated in this class's superclass, `SDLRPCMessage`.